src/ResourceManagement/CosmosDB/CosmosDBAccountImpl.cs

```diff
@@ -413,6 +413,11 @@ public CosmosDBAccountImpl WithWriteReplication(Region region, bool? isZoneRedun
             return this;
         }
 
+        public CosmosDBAccountImpl WithDefaultWriteReplication()
+        {
+            return this.WithWriteReplication(this.Region, null);
```

Comment:
Not sure whether this is needed or not. What happens if `WithWriteReplication` is skipped altogether?

Reply:
Not very well to skip it. Since the withReadReplication will do the same thing to set locations. If sb. skip write and set a read replication, it will become write replication.